survey_form.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="survey_form.css">
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id= "title" > "THE NAIROBI HOSPITAL PATIENT DATA FORM" </h1>
+    
+    <p id="description"> Patient laboratory details </p>
+    
+    <form id="survey-form" encytype= "text/plain" >
+        
+        <label for "name" id="name-label"> Name; </label>
+        <input type="text" id="name" name="name" placeholder="Enter your name here" required >
+        </br>
+        
+        <label for "email" id="email-label" > Email; </label>
+        <input type="email" id="email" name="email" placeholder="japhetcoej@gmail.com" required >
+        </br>
+    
+        <label for "number" id="number-label" > Enter your age; </label>
+        <input type="number" id="number" name="number"  min="15" max= "100" placeholder="15" >
+        </br>
+
+        <select name="dropdown" id="dropdown">
```

Comment:
Instead of using id, it would be better to use class since it is one of the best practices.

Reply:
> Instead of using id, it would be better to use class since it is one of the best practices.

- [ ] Great job here providing the review and giving a reason why a change is required, But instead of the suggestions please ask for the changes politely as required. Apply this to all of your comments.
Example: Please use class selector instead of id selector.